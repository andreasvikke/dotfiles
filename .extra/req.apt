zsh
kitty
build-essential
git
apt-transport-https
ca-certificates
curl
traceroute
tig
gnupg
lsb-release
network-manager-l2tp-gnome
rofi
picom
python3-pip
polybar
make
brave-browser
1password
1password-cli
docker-ce
docker-ce-cli
containerd.io
docker-compose-plugin
kubectl
blueman
bluez
bluez-tools
openconnect
make
gawk
npm
direnv
xdotool
gnome-tweaks
gnome-shell
gnome-shell-extensions
chrome-gnome-shell
gnome-shell-extension-manager
dbus-x11
