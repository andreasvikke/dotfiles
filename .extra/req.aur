1password-cli
kubecolor
nordic-theme-git
python-pywal16
tfswitch-bin
