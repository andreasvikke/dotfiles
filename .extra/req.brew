derailed/k9s/k9s
fzf
gcc
helm
krew
kubectx
pre-commit
terraform-docs
tflint
warrensbox/tap/tfswitch
