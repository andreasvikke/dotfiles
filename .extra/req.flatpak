flathub com.slack.Slack
flathub com.spotify.Client
