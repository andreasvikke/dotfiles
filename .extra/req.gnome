auto-move-windows@gnome-shell-extensions.gcampax.github.com
Battery-Health-Charging@maniacx.github.com
blur-my-shell@aunetx
CoverflowAltTab@palatis.blogspot.com
forge@jmmaranan.com
gnome-ui-tune@itstime.tech
pomodoro@arun.codito.in
