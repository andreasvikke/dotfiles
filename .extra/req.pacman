aws-cli
base-devel
bat
brave-browser
ca-certificates
curl
direnv
docker
docker-buildx
docker-compose
exa
file
gawk
git
gnome-browser-connector
gnome-shell
gnome-shell-extensions
gnome-tweaks
jq
kitty
otf-firamono-nerd
procps-ng
tig
tldr
traceroute
ttf-firacode-nerd
vim
zsh
