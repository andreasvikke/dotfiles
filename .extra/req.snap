code --classic
slack
notion-snap-reborn
